analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -171,99 +171,15 @@ open class Npm(
         }
 
         /**
-         * Split the given [rawName] of a module to a pair with namespace and name.
+         * Construct a [Package] by parsing its _package.json_ file and - if applicable - querying additional
```

Comment:
Commit message nits: I noticed different "Npm" vs. "NPM" spellings in this and the previous commit message. If you refer to the class, it should be "Npm", and if you refer to the tool, it should be "NPM".

Reply:
I hope I have fixed all spelling problems. It is sometimes a bit ambiguous whether the class or the tool should be referenced.